fs/ensure_dir_test.ts

```diff
@@ -11,97 +11,116 @@ Deno.test("ensureDirIfItNotExist", async function () {
   const baseDir = path.join(testdataDir, "ensure_dir_not_exist");
   const testDir = path.join(baseDir, "test");
 
-  await ensureDir(testDir);
-
-  await assertRejects(
-    async () => {
-      await Deno.stat(testDir).then(() => {
-        throw new Error("test dir should exists.");
-      });
-    },
-  );
-
-  await Deno.remove(baseDir, { recursive: true });
+  try {
+    await ensureDir(testDir);
+
+    // test dir should exists.
+    await Deno.stat(testDir);
+  } finally {
+    await Deno.remove(baseDir, { recursive: true });
+  }
 });
 
 Deno.test("ensureDirSyncIfItNotExist", function () {
   const baseDir = path.join(testdataDir, "ensure_dir_sync_not_exist");
   const testDir = path.join(baseDir, "test");
 
-  ensureDirSync(testDir);
-
-  Deno.statSync(testDir);
+  try {
+    ensureDirSync(testDir);
 
-  Deno.removeSync(baseDir, { recursive: true });
+    // test dir should exists.
+    Deno.statSync(testDir);
+  } finally {
+    Deno.removeSync(baseDir, { recursive: true });
+  }
 });
 
 Deno.test("ensureDirIfItExist", async function () {
   const baseDir = path.join(testdataDir, "ensure_dir_exist");
   const testDir = path.join(baseDir, "test");
 
-  // create test directory
-  await Deno.mkdir(testDir, { recursive: true });
+  try {
+    // create test directory
+    await Deno.mkdir(testDir, { recursive: true });
 
-  await ensureDir(testDir);
+    await ensureDir(testDir);
 
-  await assertRejects(
-    async () => {
-      await Deno.stat(testDir).then(() => {
-        throw new Error("test dir should still exists.");
-      });
-    },
-  );
-
-  await Deno.remove(baseDir, { recursive: true });
+    // test dir should still exists.
+    await Deno.stat(testDir);
+  } finally {
+    await Deno.remove(baseDir, { recursive: true });
+  }
 });
 
 Deno.test("ensureDirSyncIfItExist", function () {
   const baseDir = path.join(testdataDir, "ensure_dir_sync_exist");
   const testDir = path.join(baseDir, "test");
 
-  // create test directory
-  Deno.mkdirSync(testDir, { recursive: true });
+  try {
+    // create test directory
+    Deno.mkdirSync(testDir, { recursive: true });
 
-  ensureDirSync(testDir);
+    ensureDirSync(testDir);
 
-  assertThrows(() => {
+    // test dir should still exists.
     Deno.statSync(testDir);
-    throw new Error("test dir should still exists.");
-  });
```

Comment:
The following code seemed to be nothing tested because the error was always thrown.

```ts
assertThrows(() => {
  Deno.statSync(testDir);
  throw new Error("test dir should still exists.");
});
```

Therefore, it was rewritten as follows.

```ts
// test dir should still exists.
Deno.statSync(testDir);
```